icu_benchmarks/models/train.py

```
@@ -1,58 +1,63 @@
-    epochs=100,
-    patience=20,
-    min_delta=1e-5,
-    test_on: str = Split.test,
-    dataset_names=None,
+    precision: Optional[Literal[16] | Literal[32] | Literal[64] | Literal["16-true"]]= 32,
+    batch_size: int = 1,
+    epochs: int = 100,
+    patience: int = 20,
+    min_delta: float = 1e-5,
+    test_on: str = DataSplit.test,
+    dataset_names: dict = {},
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)